### ISSUE TITLE
Fix production log spam

### ISSUE THREAD

**Issue by gsidebo** [mitodl/bootcamp-ecommerce#1040]
While looking through production logs for an error messages, I noticed a massive amount of unnecessary log messages being logged. This increased noise makes it difficult to debug errors on prod. It seems like we just have a few templates that expect a logged-in user where one might not exist, and/or some invalid property references.

Some examples:
```
2020-10-02T17:11:13.095694+00:00 app[web.1]: [2020-10-02 17:11:13] DEBUG 49 [django.template] base.py:872 - [21dbfac8-d26d-40a1-8e91-61606c332b1b] - Exception while resolving variable 'name' in template 'home_page.html'.
2020-10-02T17:11:13.095695+00:00 app[web.1]: Traceback (most recent call last):
2020-10-02T17:11:13.095696+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 829, in _resolve_lookup
2020-10-02T17:11:13.095696+00:00 app[web.1]: current = current[bit]
2020-10-02T17:11:13.095697+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/django/utils/functional.py", line 257, in inner
2020-10-02T17:11:13.095697+00:00 app[web.1]: return func(self._wrapped, *args)
2020-10-02T17:11:13.095698+00:00 app[web.1]: TypeError: 'AnonymousUser' object is not subscriptable
2020-10-02T17:11:13.095698+00:00 app[web.1]:
2020-10-02T17:11:13.095699+00:00 app[web.1]: During handling of the above exception, another exception occurred:
2020-10-02T17:11:13.095699+00:00 app[web.1]:
2020-10-02T17:11:13.095699+00:00 app[web.1]: Traceback (most recent call last):
2020-10-02T17:11:13.095700+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 837, in _resolve_lookup
2020-10-02T17:11:13.095700+00:00 app[web.1]: current = getattr(current, bit)
2020-10-02T17:11:13.095700+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/django/utils/functional.py", line 257, in inner
2020-10-02T17:11:13.095701+00:00 app[web.1]: return func(self._wrapped, *args)
2020-10-02T17:11:13.095701+00:00 app[web.1]: AttributeError: 'AnonymousUser' object has no attribute 'name'
2020-10-02T17:11:13.095701+00:00 app[web.1]:
2020-10-02T17:11:13.095702+00:00 app[web.1]: During handling of the above exception, another exception occurred:
2020-10-02T17:11:13.095702+00:00 app[web.1]:
2020-10-02T17:11:13.095702+00:00 app[web.1]: Traceback (most recent call last):
2020-10-02T17:11:13.095703+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
2020-10-02T17:11:13.095703+00:00 app[web.1]: current = current[int(bit)]
2020-10-02T17:11:13.095704+00:00 app[web.1]: ValueError: invalid literal for int() with base 10: 'name'
2020-10-02T17:11:13.095704+00:00 app[web.1]:
2020-10-02T17:11:13.095705+00:00 app[web.1]: During handling of the above exception, another exception occurred:
2020-10-02T17:11:13.095705+00:00 app[web.1]:
2020-10-02T17:11:13.095705+00:00 app[web.1]: Traceback (most recent call last):
2020-10-02T17:11:13.095706+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 850, in _resolve_lookup
2020-10-02T17:11:13.095706+00:00 app[web.1]: (bit, current))  # missing attribute
2020-10-02T17:11:13.095751+00:00 app[web.1]: django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7f2984be68d0>>

2020-10-02T17:13:26.675630+00:00 app[web.2]: [2020-10-02 17:13:26] DEBUG 47 [django.template] base.py:872 - [49e63586-dee0-45ee-aebd-ca4c9806e04d] - Exception while resolving variable 'name' in template 'home_page.html'.
2020-10-02T17:13:26.675632+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675632+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 829, in _resolve_lookup
2020-10-02T17:13:26.675633+00:00 app[web.2]: current = current[bit]
2020-10-02T17:13:26.675633+00:00 app[web.2]: TypeError: 'HomeAlumniSection' object is not subscriptable
2020-10-02T17:13:26.675634+00:00 app[web.2]:
2020-10-02T17:13:26.675634+00:00 app[web.2]: During handling of the above exception, another exception occurred:
2020-10-02T17:13:26.675634+00:00 app[web.2]:
2020-10-02T17:13:26.675635+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675635+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 837, in _resolve_lookup
2020-10-02T17:13:26.675635+00:00 app[web.2]: current = getattr(current, bit)
2020-10-02T17:13:26.675636+00:00 app[web.2]: AttributeError: 'HomeAlumniSection' object has no attribute 'name'
2020-10-02T17:13:26.675636+00:00 app[web.2]:
2020-10-02T17:13:26.675636+00:00 app[web.2]: During handling of the above exception, another exception occurred:
2020-10-02T17:13:26.675636+00:00 app[web.2]:
2020-10-02T17:13:26.675636+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675637+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
2020-10-02T17:13:26.675637+00:00 app[web.2]: current = current[int(bit)]
2020-10-02T17:13:26.675637+00:00 app[web.2]: ValueError: invalid literal for int() with base 10: 'name'
2020-10-02T17:13:26.675637+00:00 app[web.2]:
2020-10-02T17:13:26.675638+00:00 app[web.2]: During handling of the above exception, another exception occurred:
2020-10-02T17:13:26.675638+00:00 app[web.2]:
2020-10-02T17:13:26.675638+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675638+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 850, in _resolve_lookup
2020-10-02T17:13:26.675638+00:00 app[web.2]: (bit, current))  # missing attribute
2020-10-02T17:13:26.675642+00:00 app[web.2]: django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <HomeAlumniSection: Homepage Alumni Section>
2020-10-02T17:13:26.675693+00:00 app[web.2]: [2020-10-02 17:13:26] DEBUG 47 [django.template] base.py:872 - [49e63586-dee0-45ee-aebd-ca4c9806e04d] - Exception while resolving variable 'name' in template 'home_page.html'.
2020-10-02T17:13:26.675693+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675694+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 829, in _resolve_lookup
2020-10-02T17:13:26.675694+00:00 app[web.2]: current = current[bit]
2020-10-02T17:13:26.675694+00:00 app[web.2]: TypeError: 'HomeAlumniSection' object is not subscriptable
2020-10-02T17:13:26.675694+00:00 app[web.2]:
2020-10-02T17:13:26.675695+00:00 app[web.2]: During handling of the above exception, another exception occurred:
2020-10-02T17:13:26.675695+00:00 app[web.2]:
2020-10-02T17:13:26.675695+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675695+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 837, in _resolve_lookup
2020-10-02T17:13:26.675695+00:00 app[web.2]: current = getattr(current, bit)
2020-10-02T17:13:26.675697+00:00 app[web.2]: AttributeError: 'HomeAlumniSection' object has no attribute 'name'
2020-10-02T17:13:26.675697+00:00 app[web.2]:
2020-10-02T17:13:26.675698+00:00 app[web.2]: During handling of the above exception, another exception occurred:
2020-10-02T17:13:26.675698+00:00 app[web.2]:
2020-10-02T17:13:26.675698+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675698+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
2020-10-02T17:13:26.675698+00:00 app[web.2]: current = current[int(bit)]
2020-10-02T17:13:26.675699+00:00 app[web.2]: ValueError: invalid literal for int() with base 10: 'name'
2020-10-02T17:13:26.675699+00:00 app[web.2]:
2020-10-02T17:13:26.675699+00:00 app[web.2]: During handling of the above exception, another exception occurred:
2020-10-02T17:13:26.675699+00:00 app[web.2]:
2020-10-02T17:13:26.675700+00:00 app[web.2]: Traceback (most recent call last):
2020-10-02T17:13:26.675700+00:00 app[web.2]: File "/app/.heroku/python/lib/python3.6/site-packages/django/template/base.py", line 850, in _resolve_lookup
2020-10-02T17:13:26.675701+00:00 app[web.2]: (bit, current))  # missing attribute
2020-10-02T17:13:26.675701+00:00 app[web.2]: django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <HomeAlumniSection: Homepage Alumni Section>
2020-10-02T17:13:26.677614+00:00 app[web.2]: [2020-10-02 17:13:26] DEBUG 47 [django.template] base.py:872 - [49e63586-dee0-45ee-aebd-ca4c9806e04d] - Exception while resolving variable 'name' in template 'home_page.html'.
```

**Comment by pdpinch** [comment]
Will this be fixed by #1028 ?

**Comment by gsidebo** [comment]
It will! Closing this in favor of that
